cou/apps/core.py

```diff
@@ -194,8 +194,8 @@ def channel(self, charm_channel: str) -> None:
             self._channel = charm_channel
             return
         raise ValueError(
-            f"Charm: '{self.charm}' with series: '{self.series}' has invalid track: "
-            f"'{self._get_track_from_channel(charm_channel)}'"
+            f"Track {self._get_track_from_channel(charm_channel)} is not valid for charm "
+            f"'{self.charm}' on series '{self.series}'"
```

Comment:
Thanks for updating the wording, but I think my last comment still stands: what does "invalid" means in this context? We have to try to provide operators actionable messages; what are they expected to do when they see this message?

Reply:
As we discussed today, the correct wording here is not supported. I've changed a little bit to make that clear and pointed to the OpenStack [documentaion](https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html#tracks-for-the-openstack-charms-project).